Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -135,30 +188,111 @@ void HiresTexture::Update()
       }
     }
 
-    s_textureCacheAbortLoading.Clear();
+    s_texturePrefetcher_AbortLoading.Clear();
     s_prefetcher = std::thread(Prefetch);
   }
 }
 
 void HiresTexture::Prefetch()
 {
   Common::SetCurrentThreadName("Prefetcher");
+  // bytes to MBytes calcs will auto convert stuff to double
+  // so why not use double for the rest and config so it's consistent with printf %.1f
+  // because this is still WIP - remove_this_when_done
+
+  double prefetch_starttime_total = 0;
+  double prefetch_stoptime_total = 0;
+
+  // Default 30% -- Todo GUI User Configurable, INI Only for now
+  double percent_mem_to_cull = 30.0;  // aka culled_percent
+
+  if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) != 30.0)
+  {
+    if ((Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) < 5.0) ||
+        (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0))
+    {
+      if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) < 5.0)
+      {
+        percent_mem_to_cull = 5.0;
+        OSD::AddMessage(
+            "Prefetcher: User INI setting for 'culled cercent' of " +
+                StringFromFormat(
+                    "%.1f %%", Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT)) +
+                " is below the lowest bound. Resetting to lowest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "lower bound. Setting to lowest");
+      }
+      else if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0)
+      {
+        percent_mem_to_cull = 90.0;
+        OSD::AddMessage(
+            "Prefetcher: User INI setting for 'culled percent' of " +
+                StringFromFormat(
+                    "%.1f %%", Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT)) +
+                " is below the highest bound. Resetting to highest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "higher bound. Setting to highest");
+      }
+    }
+    else
+    {
+      // Successful Manual Override - Dont report values in OSD, routine stats notification will do
+      OSD::AddMessage("Prefetcher: Maximum delegated memory limit has been manually overriden "
+                      "by user INI settings.");
```

Comment:
Doesn't that make them totally useless then? What about, we make them sacalabe, making them translated, and making them not disappear so quick?  Sometimes it better to make other things adapt to you that you adapting to them. 
But hey I forgot to add the timeout there.